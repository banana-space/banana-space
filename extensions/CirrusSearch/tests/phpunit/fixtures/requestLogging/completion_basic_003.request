{
	"type": "completion",
	"term": "Te",
	"namespaces" : [0,12]
}
