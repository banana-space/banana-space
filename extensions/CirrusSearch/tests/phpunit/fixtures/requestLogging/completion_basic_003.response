[{
    "responses": [
        {
            "took": 3,
            "timed_out": false,
            "_shards": {
                "total": 4,
                "successful": 4,
                "failed": 0
            },
            "hits": {
                "total": 0,
                "max_score": 0,
                "hits": []
            },
            "suggest": {
                "plain": [
                    {
                        "text": "Te",
                        "offset": 0,
                        "length": 2,
                        "options": [
                            {
                                "text": "Test2",
                                "_index": "wiki_titlesuggest_1504788820",
                                "_type": "titlesuggest",
                                "_id": "14t",
                                "_score": 4713,
                                "_source": {
                                    "target_title": {
                                        "namespace": 0,
                                        "title": "Test2"
                                    }
                                }
                            },
                            {
                                "text": "Test2\/en",
                                "_index": "wiki_titlesuggest_1504788820",
                                "_type": "titlesuggest",
                                "_id": "17t",
                                "_score": 132,
                                "_source": {
                                    "target_title": {
                                        "namespace": 0,
                                        "title": "Test2\/en"
                                    }
                                }
                            },
                            {
                                "text": "Test2\/fr",
                                "_index": "wiki_titlesuggest_1504788820",
                                "_type": "titlesuggest",
                                "_id": "19t",
                                "_score": 131,
                                "_source": {
                                    "target_title": {
                                        "namespace": 0,
                                        "title": "Test2\/fr"
                                    }
                                }
                            },
                            {
                                "text": "Test\/Subpage 1",
                                "_index": "wiki_titlesuggest_1504788820",
                                "_type": "titlesuggest",
                                "_id": "20t",
                                "_score": 78,
                                "_source": {
                                    "target_title": {
                                        "namespace": 0,
                                        "title": "Test\/Subpage 1"
                                    }
                                }
                            }
                        ]
                    }
                ],
                "plain_stop": [
                    {
                        "text": "Te",
                        "offset": 0,
                        "length": 2,
                        "options": [
                            {
                                "text": "Test2",
                                "_index": "wiki_titlesuggest_1504788820",
                                "_type": "titlesuggest",
                                "_id": "14t",
                                "_score": 4713,
                                "_source": {
                                    "target_title": {
                                        "namespace": 0,
                                        "title": "Test2"
                                    }
                                }
                            },
                            {
                                "text": "Test2\/en",
                                "_index": "wiki_titlesuggest_1504788820",
                                "_type": "titlesuggest",
                                "_id": "17t",
                                "_score": 131,
                                "_source": {
                                    "target_title": {
                                        "namespace": 0,
                                        "title": "Test2\/en"
                                    }
                                }
                            },
                            {
                                "text": "Test2\/fr",
                                "_index": "wiki_titlesuggest_1504788820",
                                "_type": "titlesuggest",
                                "_id": "19t",
                                "_score": 131,
                                "_source": {
                                    "target_title": {
                                        "namespace": 0,
                                        "title": "Test2\/fr"
                                    }
                                }
                            },
                            {
                                "text": "Test\/Subpage 1",
                                "_index": "wiki_titlesuggest_1504788820",
                                "_type": "titlesuggest",
                                "_id": "20t",
                                "_score": 78,
                                "_source": {
                                    "target_title": {
                                        "namespace": 0,
                                        "title": "Test\/Subpage 1"
                                    }
                                }
                            }
                        ]
                    }
                ]
            },
            "status": 200
        },
        {
            "took": 2,
            "timed_out": false,
            "_shards": {
                "total": 4,
                "successful": 4,
                "failed": 0
            },
            "hits": {
                "total": 1,
                "max_score": 3.5431085,
                "hits": [
                    {
                        "_index": "wiki_general_1504788773",
                        "_type": "page",
                        "_id": "34",
                        "_score": 3.5431085,
                        "_source": {
                            "namespace_text": "Help",
                            "wiki": "wiki",
                            "namespace": 12,
                            "title": "Test"
                        },
                        "highlight": {
                            "title.prefix_asciifolding": [
                                "<span class=\"searchmatch\">Te<\/span>st"
                            ],
                            "title.prefix": [
                                "<span class=\"searchmatch\">Te<\/span>st"
                            ]
                        }
                    }
                ]
            },
            "status": 200
        }
    ]
}]
