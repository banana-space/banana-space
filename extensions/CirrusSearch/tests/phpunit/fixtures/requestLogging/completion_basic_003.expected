[
    {
        "channel": "cirrussearch-request",
        "context": {
            "$schema": "\/mediawiki\/cirrussearch\/request\/0.0.1",
            "all_elasticsearch_requests_cached": false,
            "elasticsearch_requests": [
                {
                    "cached": false,
                    "hits": [
                        {
                            "index": "wiki_titlesuggest",
                            "page_id": 14,
                            "page_title": "Test2",
                            "profile_name": "plain",
                            "score": 4713
                        },
                        {
                            "index": "wiki_titlesuggest",
                            "page_id": 17,
                            "page_title": "Test2\/en",
                            "profile_name": "plain",
                            "score": 132
                        },
                        {
                            "index": "wiki_titlesuggest",
                            "page_id": 19,
                            "page_title": "Test2\/fr",
                            "profile_name": "plain",
                            "score": 131
                        },
                        {
                            "index": "wiki_titlesuggest",
                            "page_id": 20,
                            "page_title": "Test\/Subpage 1",
                            "profile_name": "plain",
                            "score": 78
                        },
                        {
                            "index": "wiki_general",
                            "page_id": 34,
                            "page_title": "Help:Test",
                            "profile_name": "prefix",
                            "score": 77
                        }
                    ],
                    "hits_offset": 0,
                    "hits_returned": 5,
                    "hits_total": 9,
                    "indices": [
                        "wiki_titlesuggest",
                        "wiki_general"
                    ],
                    "max_score": 4713,
                    "namespaces": [
                        0,
                        12
                    ],
                    "query": "Te",
                    "query_type": "comp_suggest",
                    "search_time_ms": 0,
                    "suggestion_requested": false
                }
            ],
            "http": {
                "client_ip": "127.0.0.1",
                "has_cookies": false,
                "method": "GET"
            },
            "source": "cli"
        },
        "level": "debug",
        "message": ""
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 5,
            "hitsTotal": 9,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "wiki_titlesuggest,wiki_general",
            "maxScore": 4713,
            "query": "Te",
            "queryType": "comp_suggest",
            "source": "cli",
            "tookMs": 0
        },
        "level": "debug",
        "message": "{queryType} search for '{query}' against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset}. Requested via {source} for {identity} by executor {executor}"
    }
]